### PR TITLE
Modal styling

### PR DESCRIPTION
**Pull request by KennethEllebring**
- added teanary string for choosing "like" vs "likes" if its more than 1 like, also removed header from post modal
- added a few classnames and changed some in edit modal
- added stylcheet path to feed.js
- added scss code for post/edit modals

### CODE REVIEW

File: src/styles/Popup_modal.scss [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+      }
+    }
+
+    form.new-comment {
+      position: fixed;
+      top: auto;
+      bottom: 18%;
+      display: flex;
+      margin-bottom: 1rem;
+      .new-comment-input {
```
Comment (arapg) [inline src/styles/Popup_modal.scss:87]: Maybe add in `border: none;`?

File: src/styles/Popup_modal.scss [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+        background-color: #ff0000;
+      }
+    }
+    .edit-post-header {
+      font-weight: bold;
+      text-decoration: none;
+      color: #ffffff;
+      margin-bottom: 1rem;
+    }
+    .edit-comment {
```
Comment (arapg) [inline src/styles/Popup_modal.scss:158]: Shouldn't it be `.edit-post`? ;-)

Comment (KennethEllebring) [reply src/styles/Popup_modal.scss]: changed names to match edit post instead of comment

File: src/styles/Popup_modal.scss (hunk 1 above)
Comment (KennethEllebring) [reply src/styles/Popup_modal.scss:106]: removed borders on both edit posts and post comments

**Comment by KennethEllebring** [comment]
- changed the ugly windows scroll with a newer modern look
- changed the scroll to only scroll comments
- changed some classnames that were confusing
- moved the input comment inside comment post window

**Review by AlexSaxena — APPROVED** [review]
Snyggt jobbat! Gillar nya scrollbaren 👌

**Pull request merged**